<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-10-21T00:09:26Z" name="Regression Test Suite" finished-at="2020-10-21T00:09:41Z" duration-ms="15071">
    <groups>
      <group name="smoke">
        <method signature="TestClass.loginVerify()[pri:1, instance:testClasses.TestClass@2a65bb85]" name="loginVerify" class="testClasses.TestClass"/>
        <method signature="TestClass.regressionTest1()[pri:3, instance:testClasses.TestClass@2a65bb85]" name="regressionTest1" class="testClasses.TestClass"/>
        <method signature="TestClass.regressionTest2()[pri:4, instance:testClasses.TestClass@2a65bb85]" name="regressionTest2" class="testClasses.TestClass"/>
        <method signature="TestClass.verifyAboutUSpage()[pri:2, instance:testClasses.TestClass@2a65bb85]" name="verifyAboutUSpage" class="testClasses.TestClass"/>
      </group> <!-- smoke -->
    </groups>
    <test started-at="2020-10-21T00:09:26Z" name="Application Test" finished-at="2020-10-21T00:09:41Z" duration-ms="15071">
      <class name="testClasses.TestClass">
        <test-method is-config="true" signature="commonSetUp(java.lang.String)[pri:0, instance:testClasses.TestClass@2a65bb85]" started-at="2020-10-21T00:09:26Z" name="commonSetUp" finished-at="2020-10-21T00:09:35Z" duration-ms="8895" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- commonSetUp -->
        <test-method signature="loginVerify()[pri:1, instance:testClasses.TestClass@2a65bb85]" started-at="2020-10-21T00:09:35Z" name="loginVerify" finished-at="2020-10-21T00:09:37Z" duration-ms="1828" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [] but found [Warning: No match for E-Mail Address and/or Password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [] but found [Warning: No match for E-Mail Address and/or Password.]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at testClasses.TestClass.loginVerify(TestClass.java:46)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginVerify -->
        <test-method signature="verifyAboutUSpage()[pri:2, instance:testClasses.TestClass@2a65bb85]" started-at="2020-10-21T00:09:37Z" name="verifyAboutUSpage" finished-at="2020-10-21T00:09:40Z" duration-ms="3312" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyAboutUSpage -->
        <test-method signature="regressionTest1()[pri:3, instance:testClasses.TestClass@2a65bb85]" started-at="2020-10-21T00:09:40Z" name="regressionTest1" finished-at="2020-10-21T00:09:40Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- regressionTest1 -->
        <test-method signature="regressionTest2()[pri:4, instance:testClasses.TestClass@2a65bb85]" started-at="2020-10-21T00:09:40Z" name="regressionTest2" finished-at="2020-10-21T00:09:40Z" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- regressionTest2 -->
        <test-method is-config="true" signature="commonTearDown()[pri:0, instance:testClasses.TestClass@2a65bb85]" started-at="2020-10-21T00:09:40Z" name="commonTearDown" finished-at="2020-10-21T00:09:41Z" duration-ms="609" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- commonTearDown -->
      </class> <!-- testClasses.TestClass -->
    </test> <!-- Application Test -->
  </suite> <!-- Regression Test Suite -->
</testng-results>
